content/push/configure/web.textile

```diff
@@ -0,0 +1,153 @@
+const client = new Ably.Realtime({
+  pushServiceWorkerUrl: '/service_worker.js',
+  plugins: { Push }
+})
+```
+
+h2. Activate browers
+
+To receive push notifications using Ably, each browser must first register with the service worker registration's push manager for web push notifications. Ably's SDK facilitates this registration process and provides a unified API to manage it.
+
+<a href="@content/diagrams/push-process-activate-highlighted-web-push.png" target="_blank">
+  <img src="@content/diagrams/push-process-activate-highlighted-web-push.png" style="width: 100%" alt="Push Notifications in Ably">
+</a>
+
+h3(#browser). Activate directly
+
+Activating a browser for push notifications is typically done directly on the browser itself. Use the "@push.activate()@":/api/realtime-sdk/push#activate method to activate push notifications from a browser. The following steps describe the direct activation process:
+
+# Authenticate the Ably client.
+# Generate a unique identifier for the browser and saves it locally.
+# Activate push notifications for the browers with the service worker registration's push manager, obtaining a unique identifier.
+# Register the browser with Ably, providing its unique identifier (@deviceId@), browser details, and push recipient information.
+# Store the browser's identity token from Ably's response locally for authentication in subsequent updates.
+
+Note that once activated, the browser remains registered even after the application is closed until the "@push.deactivate()@":/api/realtime-sdk/push#deactivate method is called. Calling activate again has no effect if the browser is already activated.
+
+The following diagram demonstrates the direct activation process:
+
+<a href="@content/diagrams/push-direct-activation-web-push.png" target="_blank">
+  <img src="@content/diagrams/push-direct-activation-web-push.png" style="width: 100%" alt="push notifications in Ably">
+</a>
+
+The following example initializes an instance of the Ably realtime service and then activates the push notifications feature _for that instance_:
+
+```[javascript]
+await ably.push.activate();
+```
+
+h4. Test your push notification activation
+
+* Use the Ably "dashboard":https://ably.com/accounts or "API":/api/realtime-sdk/push-admin to send a test push notification to a registered browser.
+* Ensure your application correctly receives and handles the push notification.
+
+h3(#server). Activate browsers via server
+
+In specific scenarios, especially where strict control over browser capabilities is essential, managing the activation of browsers for push notifications through the server is useful. This approach separates the registration with the Web Push service, which still happens on the browser. Meanwhile, your server manages the browsers registration with Ably. This setup provides a centralized and efficient process for controlling browser activation.
```

Comment:
@owenpearson is this a thing? I thought you said a while back that we weren't supporting remote registration

Reply:
yep it is possible. i can't remember why but at one point i thought it wasn't possible/worthwhile but it's simple enough to support so yeah it is a thing.